vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt

```diff
@@ -100,8 +99,6 @@ class CallAudioManager(
         val audioManager = audioManager ?: return
         savedIsSpeakerPhoneOn = audioManager.isSpeakerphoneOn
         savedIsMicrophoneMute = audioManager.isMicrophoneMute
-        savedAudioMode = audioManager.mode
```

Comment:
We don't have to restore the mode anymore? If i was listening to music on speaker before the call what happens after?

Reply:
During my tests it was ok. I guess the app will get focus again